.doctrine-project.json

```
@@ -4,16 +4,22 @@
   "slug": "doctrine-bundle",
   "versions": [
     {
-      "name": "2.11",
-      "branchName": "2.11.x",
+      "name": "2.12",
+      "branchName": "2.12.x",
       "slug": "latest",
       "upcoming": true
     },
+    {
+      "name": "2.11",
+      "branchName": "2.11.x",
+      "slug": "2.11",
+      "aliases": ["current", "stable"]
```

Should we take this as an occasion to simplify?



```suggestion
      "current": true
```

See https://github.com/doctrine/doctrine-website/blob/998fbbc30d0731fe97f727f05dde832e2fc4286e/lib/Model/ProjectVersion.php#L68-L73